src/embeddedWebserver.h

```diff
@@ -86,72 +52,91 @@ String getTempString() {
 }
 
 // proper modulo function (% is remainder, so will return negatives)
-int mod(int a, int b) {
-    int r = a % b;
+inline int mod(const int a, const int b) {
+    const int r = a % b;
     return r < 0 ? r + b : r;
 }
 
 // rounds a number to 2 decimal places
 // example: round(3.14159) -> 3.14
 // (less characters when serialized to json)
-double round2(double value) {
-    return (int)(value * 100 + 0.5) / 100.0;
+inline double round2(const double value) {
+    return static_cast<int>(value * 100 + 0.5) / 100.0;
 }
 
-String getValue(String varName) {
+inline String getValue(const String& varName) {
     try {
-        editable_t e = editableVars.at(varName);
-        switch (e.type) {
-            case kFloat:
-                return String(*(float*)e.ptr);
-            case kDouble:
-                return String(*(double*)e.ptr);
-            case kDoubletime:
-                return String(*(double*)e.ptr);
-            case kInteger:
-                return String(*(int*)e.ptr);
-            case kUInt8:
-                return String(*(uint8_t*)e.ptr);
-            case kCString:
-                return String((char*)e.ptr);
-            default:
-                return F("Unknown type");
-                break;
+        const std::shared_ptr<Parameter> e = ParameterRegistry::getInstance().getParameterById(varName.c_str());
```

Comment:
```suggestion
        const auto e = ParameterRegistry::getInstance().getParameterById(varName.c_str());
```

You don't have to type your fingers to the bone. :wink: